xbmc/addons/addoninfo/AddonInfoBuilder.h

```diff
@@ -56,8 +56,8 @@ class CAddonInfoBuilder
     }
     void SetPath(std::string path) { m_addonInfo->m_path = std::move(path); }
     void SetLibName(std::string libname) { m_addonInfo->m_libname = std::move(libname); }
-    void SetVersion(const AddonVersion& version) { m_addonInfo->m_version = version; }
-    void SetMinVersion(const AddonVersion& minversion) { m_addonInfo->m_minversion = minversion; }
+    void SetVersion(const AddonVersion& version) { m_addonInfo->m_version = std::move(version); }
+    void SetMinVersion(const AddonVersion& minversion) { m_addonInfo->m_minversion = std::move(minversion); }
```

Comment:
...but as i see it now a few seconds too late: this cannot work as long as we pass in by `const &`, right?